src/server/database/Database/PreparedStatement.cpp

```diff
@@ -33,54 +33,55 @@ void PreparedStatement::BindParameters(MySQLPreparedStatement* stmt)
     ASSERT(stmt);
     m_stmt = stmt;
 
-    uint8 i = 0;
-    for (; i < statement_data.size(); i++)
+    uint8 pos = 0;
+    for (PreparedStatementData data : statement_data)
```

Comment:
is this another case of "copy" ?

Reply:
That should've been fixed already, but yes 😅